.mergify.yml

```diff
@@ -151,6 +151,31 @@ pull_request_rules:
         strict: smart
       dismiss_reviews: {}
       delete_head_branch: {}
+  - name: backport patches to release-v3.1 branch
+    conditions:
+      - base=master
+      - label=backport-to-release-v3.1
+    actions:
+      backport:
+        branches:
+          - release-v3.1
+  # automerge backports if CI successfully ran
+  - name: automerge backport release-v3.1
+    conditions:
+      - author=mergify[bot]
+      - base=release-v3.1
+      - label!=DNM
+      - "#changes-requested-reviews-by=0"
+      - "#approved-reviews-by>=1"
+      - "status-success=continuous-integration/travis-ci/pr"
```

Comment:
should we not require the ci/centos/* status too?

Reply:
Yes, I have copy-pasted from the older release will update it